homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+  padding-bottom: 40px;
+  &-item-text{
+    padding: 0 20px;
+  }
+  &-text{
+    margin-top: 20px;
+    font-size: 13px;
+    line-height: 24px;
+  }
+  &-img{
+    margin: 30px auto 0;
+  }
+}
+.get-more{
+  margin-top: -60px;
+  padding: 19px 25px 30px;
+  background-image: $img-bg-card;
+  border-radius: 5px;
+  filter: drop-shadow(0px 12px 24px #e5e5e5);
+  &-title{
+    padding-bottom: 5px;
+    color: $color-get-more-title;
+    font-family: $font-title, sans-serif;
+    font-size: 10px;
+    line-height: 24px;
+    letter-spacing: 1.6px;
+    text-transform: uppercase;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.5);
+  }
+  &-link{
+    margin: 34px 0 0 0px;   
+    .get-link{
+      color: $color-get-more-title;
+      font-size: 16px;
+      line-height: 24px;
+    }
+  }
+  .get-item{
+    margin-top: 40px;
+  }
+}
+.get-text{
+  position: relative;
+  &::after{
+    content: '';
+    position: absolute;
+    display: inline-block;
+    width: 35px;
+    height: 35px;
+    margin: -7px 0 0 89px;
+    background-color: transparent;
+    border: 1px solid #fff;
+    border-radius: 50%;
+  }
+}
+.card-item-wrapper{
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: 9px 0 16px;
+  &::before{
+    content: "";
+    position: absolute;
+    top: 0;
+    width: 218px;
+    height: 2px;
+    background: #f2f2f2;
+  }
+}
+.card-item{
+  position: relative;
+  margin-left: 20px;
+  color: $color-description-text;
+  font-size: 13px;
+  line-height: 24px;
+  &-minus{
+    color: #e5e5e5;
+  }
+}
+.check-icon{
+  position: relative;
+  display: inline-block;
+  width: 35px;
+  height: 35px;
+  background: transparent;
+  border: 2px solid #22bf95;
+  border-radius: 50%;
+  &::before{
+    content: "";
+    width: 2px;
+    background: #22bf95;
+    height: 9px;
+    position: absolute;
+    top: 14px;
+    transform: rotate(138deg);
+    left: 9px;
+  }
+  &::after{
+    content: "";
+    width: 2px;
+    background: #22bf95;
+    height: 18px;
+    position: absolute;
+    top: 7px;
+    transform: rotate(-138deg);
+    left: 18px;
+  }
+}
+.minus-icon{
+  position: relative;
+  display: inline-block;
+  width: 35px;
+  height: 35px;
+  background: transparent;
+  border: 2px solid #e5e5e5;
+  border-radius: 50%;
+  &::before{
+    content: "";
+    width: 18px;
+    background: #e5e5e5;
+    height: 2px;
+    position: absolute;
+    top: 14px;
+    left: 8px;
+  }
+}
+.arrow{
+  height: 13px;
+  position: relative;
+  width: 1px;
+  background: #fff;
+  margin: 4px 16px 0 100px;
+  display: inline-block;
+  border: 1px solid #fff;
+  &:before{
+    content: "";
+    width: 2px;
+    background: #fff;
+    height: 10px;
+    position: absolute;
+    top: -5px;
+    transform: rotate(50deg);
+    left: -4px;
+  }
+  &:after{
+    content: "";
+    width: 2px;
+    background: #fff;
+    height: 10px;
+    position: absolute;
+    top: -5px;
+    transform: rotate(-50deg);
+    right: -4px;
+  }
+}
+.right{
```

Comment:
this class can be used only with `.arrow` class.
So you have to write `.arrow.right` or using scss:
```
.arrow {
  ...
  .right {
    ...
  }
}
```

Reply:
done